### PR TITLE
[PPP-4283] Use of Vulnerable Components: commons-httpclient-3.0.1.jar…

### PR DESCRIPTION
**Pull request by ppatricio**
… and httpclient-4.0.1.jar (sonatype-2007-0004 | CVE-2011-1498 | CVE-2012-6153 | CVE-2012-5783)

### CODE REVIEW

File: kettle-sdk-embedding-samples/pom.xml [hunk 1]
```diff
@@ -43,7 +43,10 @@
     <dependency>
       <groupId>org.apache.httpcomponents</groupId>
       <artifactId>httpclient</artifactId>
-      <version>4.5.3</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.httpcomponents</groupId>
+      <artifactId>httpcore</artifactId>
```
Comment (graimundo) [inline kettle-sdk-embedding-samples/pom.xml]: Check if scope is test

**Comment by buildguy** [comment]
### :x: Build failed in 32s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl kettle-sdk-embedding-samples
```
---
 :exclamation: No tests found!
 ---
### Errors:
<details><summary><b>Filtered log</b> (click to expand)</summary>
<p>

```
script returned exit code 1
```
 
</p>
</details>
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :x: Build failed in 1m
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl kettle-sdk-embedding-samples
```
---
 :exclamation: No tests found!
 ---
### Errors:
<details><summary><b>Filtered log</b> (click to expand)</summary>
<p>

```
script returned exit code 1
```
 
</p>
</details>
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :x: Build failed in 20s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl kettle-sdk-embedding-samples
```
---
 :exclamation: No tests found!
 ---
### Errors:
<details><summary><b>Filtered log</b> (click to expand)</summary>
<p>

```
script returned exit code 1
```
 
</p>
</details>
 
 

:information_source:  _This is an automatic message_

File: kettle-sdk-embedding-samples/pom.xml (hunk 1 above)
Comment (ppatricio) [reply kettle-sdk-embedding-samples/pom.xml]: Not only required for tests. It's being used also.

**Pull request merged**